cylc/flow/network/schema.py

```diff
@@ -867,330 +905,681 @@ class Meta:
-        event_message = String(required=True)
-        event_id = String(required=True)
+        workflows = List(
+            WorkflowID,
+            required=True
+        )
+        tasks = List(
+            NamespaceIDGlob,
+            required=True
+        )
 
     result = GenericScalar()
 
 
+class DryRun(Mutation, TaskMutation):
+    class Meta:
+        description = sstrip('''
+            [For internal use] Prepare the job file for a task.
+        ''')
+        resolver = partial(mutator, command='dry_run_tasks')
+
+    class Arguments(TaskMutation.Arguments):
+        check_syntax = Boolean(
+            description='Check shell syntax.',
+            default_value=True
+        )
+
+
+class Insert(Mutation, TaskMutation):
+    class Meta:
+        description = sstrip('''
+            Insert new task proxies into the task pool of a running workflow.
+
+            For example to enable re-triggering earlier tasks already removed
+            from the pool.
+
+            Note: inserted cycling tasks cycle on as normal, even if another
+            instance of the same task exists at a later cycle (instances of the
+            same task at different cycles can coexist, but a newly spawned task
+            will not be added to the pool if it catches up to another task with
+            the same ID).
+
+            See also "Submit", for running tasks without the scheduler.
+        ''')
+        resolver = partial(mutator, command='insert_tasks')
+
+    class Arguments(TaskMutation.Arguments):
+        check_point = Boolean(
+            description=sstrip('''
+                Check that the provided cycle point is on one of the task's
+                recurrences as defined in the suite configuration before
+                inserting.
+            '''),
+            default_value=True
+        )
+        stop_point = CyclePoint(
+            description='hold/stop cycle point for inserted task.'
+        )
+
+
+class Kill(Mutation, TaskMutation):
+    # TODO: This should be a job mutation?
+    class Meta:
+        description = sstrip('''
+            Kill jobs of active tasks and update their statuses accordingly.
+        ''')
+        resolver = partial(mutator, command='kill_tasks')
+
+
+class Poll(Mutation, TaskMutation):
+    class Meta:
+        description = sstrip('''
+            Poll (query) task jobs to verify and update their statuses.
+        ''')
+        resolver = partial(mutator, command='poll_tasks')
+
+    class Arguments(TaskMutation.Arguments):
+        poll_succeeded = Boolean(
+            description='Allow polling of succeeded tasks.',
+            default_value=False
+        )
+
+
+class Remove(Mutation, TaskMutation):
+    class Meta:
+        description = sstrip('''
+            Remove one or more task instances from a running workflow.
+
+            Tasks will be forced to spawn successors before removal if they
+            have not done so already, unless you use `no_spawn`.
```

Comment:
Here it says `no_spawn`. Under the `Arguments` class below, it has a `spawn = Boolean`... is that correct?